### PR TITLE
SnackTV: remove wildcard

### PR DESCRIPTION
**Pull request by numismatika**

### CODE REVIEW

**Comment by J0WI** [comment]
Could you please also add ^snacktv.de?

File: src/chrome/content/rules/Snacktv.de.xml [hunk 1]
```diff
@@ -1,5 +1,13 @@
...
-<target host="*.snacktv.de"/>
 
-<rule from="^http://(player|www)\.snacktv\.de/" to="https://$1.snacktv.de/"/>
+	<target host="cache.snacktv.de"/>
+	<target host="displayomat.snacktv.de"/>
+	<target host="events.snacktv.de"/>
+	<target host="lists.snacktv.de"/>
+	<target host="monetizer.snacktv.de"/>
+	<target host="player.snacktv.de"/>
+	<target host="www.snacktv.de"/>
```
Comment (J0WI) [inline src/chrome/content/rules/Snacktv.de.xml]: snackomat.snacktv.de
snackomat2.snacktv.de

**Comment by numismatika** [comment]
that happens when you code from memory instead of copying the original rule file i wrote..
Will re-add them shortly.

**Comment by J0WI** [comment]
This still misses ^snacktv.de

**Comment by numismatika** [comment]
@J0WI  is an incomplete certificate chain ok or should i leave it out now?

**Comment by J0WI** [comment]
Oh, I missed that issue. Thanks for updating!

**Pull request merged**